packages/tupaia-web/src/layout/TopBar/Logo.tsx

```diff
@@ -79,7 +79,7 @@ export const Logo = ({ logoSrc, displayName = false, name = '' }: LogoProps) =>
         {/** If a custom landing page has set to display the name in the header, display it here */}
         {displayName && (
           <NameWrapper>
-            <Name variant="h1">{name}</Name>
+            <Name variant="h6">{name}</Name>
```

Comment:
The way that heading levels work is that they are by hierarchy, and not by size. They naturally have different sizes, but it's not considered good practice to use a particular heading level for styling purposes. You should always have the correct order, and not have, for example, `h6` without an `h1`-`h5` ahead of it. In this case, we want it to be `h1` because it is the page title. 
Instead, just setting the font size on the `Name` component would be better.